tidy3d/components/medium.py

```diff
@@ -3318,16 +3545,29 @@ def eps_dataarray_freq(
 class AnisotropicMedium(AbstractMedium):
     """Diagonally anisotropic medium.
 
-    Note
-    ----
-    Only diagonal anisotropy is currently supported.
+    Notes
+    -----
```

Comment:
newline after this?

Reply:
Fixed in this file but will review all the rest during the full review